### PR TITLE
Create associatr_runner.py

### PR DESCRIPTION
**Pull request by hopedisastro**
This script runs associaTR, given a chromosome or cell type.
Ensure prior scripts have been run to generate dependent files, particularly:
- get_cis_numpy_files.py
- pseudobulk.py
- get_covariates.py
- qc_filters_associatr.py (depends on qc_annotator.py)

Successful batch - https://batch.hail.populationgenomics.org.au/batches/434543/jobs/1 (this now means the pipeline can run end to end)

### CODE REVIEW

**Comment by hopedisastro** [comment]
the input vcf currently is a genome-wide VCF. If we break apart the VCF into chr-specific, we could reduce the storage req per job and potentially save quite a bit when we add up all the jobs we'll be doing (#genes x # cell types)

File: str/associatr/associatr_runner.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    max_parallel_jobs,
+    cis_window_size,
+    vcf_file_dir,
+    gene_list_dir,
+    cis_window_dir,
+    pheno_cov_numpy_dir,
+    version,
+    job_storage,
+    job_cpu,
+):
```
Comment (MattWellie) [inline str/associatr/associatr_runner.py:73]: before merging this did you want to transition these arguments to config?

File: str/associatr/associatr_runner.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[associatr]
+celltypes='CD8_TEM'
+chromosomes='chr1'
+# gs://... to the chr-specific output of qc_filters_associatr.py
+vcf_file_dir='gs://cpg-bioheart-test/str/associatr/input_files/vcf/v1-chr-specific' \
+gene_list_dir='gs://cpg-bioheart-test/str/associatr/rna_pc_calibration/2_pcs/input_files/scRNA_gene_lists/1_min_pct_cells_expressed' \
+cis_window_dir='gs://cpg-bioheart-test/str/associatr/rna_pc_calibration/2_pcs/input_files/cis_window_files/v1' \
```
Comment (MattWellie) [inline str/associatr/associatr_runner.toml]:
```suggestion
# gs://... to the chr-specific output of qc_filters_associatr.py
vcf_file_dir='gs://cpg-bioheart-test/str/associatr/input_files/vcf/v1-chr-specific'
gene_list_dir='gs://cpg-bioheart-test/str/associatr/rna_pc_calibration/2_pcs/input_files/scRNA_gene_lists/1_min_pct_cells_expressed'
cis_window_dir='gs://cpg-bioheart-test/str/associatr/rna_pc_calibration/2_pcs/input_files/cis_window_files/v1'
```

**Review by MattWellie — APPROVED** [review]
that one change is required in the TOML, but looks good

**Pull request merged**